gammapy/data/observations.py

```diff
@@ -342,6 +382,61 @@ def select_time(self, time_interval):
         obs.obs_filter = new_obs_filter
         return obs
 
+    @classmethod
+    def from_file(cls, event_file, irf_file):
```

Comment:
Here I'm leaning towards just calling this `Observation.read()`, I think `.from_file()` is only used in `DataStore.from_file()`, but for all the other classes in Gammapy we use `.read()`.